src/xrGame/quadtree_inline.h

```diff
@@ -102,6 +114,11 @@ IC void CSQuadTree::insert(_object_type* object)
         if (!*node)
             *node = m_nodes->get_object();
 
+        if (node == NULL)
```

Comment:
`node` gets dereferenced 3 lines above :D
Also, we are in the cycle here. I'm not sure at the moment if we should return or continue the cycle.

Reply:
Now get_object() can return null, needs to check for it.

Reply:
Based on code already there return is ok, but will check with continue.

Reply:
> Now get_object() can return null, needs to check for it.

But here we check if `node` is null, while `get_object` assigns to `*node`, not `node` 🤔

Reply:
> > Now get_object() can return null, needs to check for it.
> 
> But here we check if `node` is null, while `get_object` assigns to `*node`, not `node` 🤔



> > Now get_object() can return null, needs to check for it.
> 
> But here we check if `node` is null, while `get_object` assigns to `*node`, not `node` 🤔

node is CQuadNode** node so *node is pointer I think? Otherwise it wouldn't compile.